simulator/setting_up_simulator.md

```diff
@@ -48,50 +62,49 @@ The simulation for the SR2024 competition will be released at Kickstart.
+On recent versions of macOS you may need to give Webots permission to access the directory where you have extracted the simulation files.
 </div>
 
+
 ### Changing your version of Python
 
-If webots is not picking any version of Python or is picking up the wrong one then you'll need to change it.
+Sometimes Webots will not automatically detect your installed Python so we need to change it.
 When this happens Webots will print errors to its console and your robot will not move.
 
 You will need the full path to the version of Python that you want to use.
 This will vary based on the system you have.
-One way to find the path is by launching the version of Python that you want to
-use and running the following code:
+One way to find the path is by launching Python and running the following code:
 
 ~~~~~ python
 import sys
 print(sys.executable)
 ~~~~~
 
-On Windows you can set the path to the Python version to use in Webots UI via
-**Tools** &rarr; **Preferences** &rarr; **General** &rarr; **Python command**.
+Once you have the path you need to enter this into the Webots settings.
+To do this on Windows, open webots and go to the menu **Tools** &rarr; **Preferences** &rarr; **General** &rarr; **Python command** and enter the path in that box.
 Your Python path is likely similar to `C:\Users\<USERNAME>\AppData\Local\Programs\Python\Python311\python.exe` when using Python 3.11, where `<USERNAME>` is your login.
 
 On Mac you can set the path to the Python version to use via **Webots** &rarr; **Preferences** <kbd>⌘</kbd><kbd>,</kbd>.
 Your Python path is likely similar to `/Library/Frameworks/Python.framework/Versions/3.11/bin/python3` when using Python 3.11.
 
-You'll need to ensure a matching version of Python is installed. If you're still
-having problems, ask for help in [`#simulator-help`][simulator-help] in
-[Discord][discord].
+If you're still having problems, ask for help in [Discord][discord].
+
 
 ## Updates
 
-Occasionally, we may release an update to the simulation. To update, you will need to delete the `competition-simulator-<version>` directory, and re-download it using the above link.
+Occasionally, we may release an update to the simulation.
+To update, you will need to delete the `competition-simulator-<version>` folder and replace it with the new version that can be downloaded using the above link.
 
-If you need a specific version of the simulator, or want to see what changes
-have been made with each version, please see the
-[list of releases](https://github.com/srobo/competition-simulator/releases).
+If you need a specific version of the simulator, or want to see what changes have been made with each version, please see the [list of releases][release-list].
 
 [discord]: {{ site.baseurl }}/tutorials/discord
 [programming-help]: https://discord.com/channels/900501415548579842/900501416269971457
 [simulator-help]: https://discord.com/channels/900501415548579842/900501416269971458
```

Comment:
These links are no longer used after these changes (and from SR2022!), please could we remove them?

Reply:
#1b1c6a7